internal/server/evaluation/legacy_evaluator.go

```diff
@@ -251,7 +251,8 @@ func matchConstraints(evalCtx map[string]string, constraints []storage.Evaluatio
 		}
 
 		if err != nil {
-			return false, reason, err
+			e.logger.Info("error matching constraint", zap.String("property", c.Property), zap.Error(err))
```

Comment:
Maybe debug. This could print _a lot_ if they're evaluating at a high frequency.